x-pack/plugins/uptime/server/lib/requests/get_monitor_status.ts

```diff
@@ -121,34 +137,65 @@ export const getMonitorStatus: UMElasticsearchQueryFn<
             fields: {
               top_hits: {
                 size: 1,
+                sort: [{ '@timestamp': 'desc' }],
```

Comment:
So the most recent document is guaranteed to have `url.full`, because it should be the `heartbeat/summary`?

Reply:
Yes, that's Indeed true

Reply:
Seems mostly right to me, we do include the URL. However, there may not be a URL if the monitor totally fails. Actually, in that case we have no summary, but we aim to fix that bug. See https://github.com/elastic/beats/issues/28770